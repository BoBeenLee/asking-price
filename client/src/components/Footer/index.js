import React from "react";
import styled from 'react-emotion';

const Root = styled('div') `
    height: 150px;
    background: ${props => props.theme.backgroundColor};
    opacity: 0.7;
`;

const Footer = () => {
    return (
        <Root />
    );
};

export default Footer;