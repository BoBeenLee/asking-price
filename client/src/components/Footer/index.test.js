import React from 'react';
import renderer from 'react-test-renderer';

import Footer from './';

describe('footer', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<Footer />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});