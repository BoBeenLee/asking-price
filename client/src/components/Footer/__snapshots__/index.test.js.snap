// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer renders correctly 1`] = `
<div
  className="css-1hcg4a0 eyr31qi0"
/>
`;
